view/frontend/templates/js/boltjs.phtml

```diff
@@ -90,3 +91,12 @@ if ($isLoadBoltJs && $isLoadConnectJs) {
         data-publishable-key="<?= /* @noEscape */ $checkoutKey; ?>">
     </script>
 <?php endif; ?>
+<?php if ($isShopperAssistantEnabledJs): ?>
```

Comment:
We want to have this script in all pages, right?

Reply:
Yup!